bsi/src/main/java/org/roaringbitmap/bsi/BitmapSliceIndex.java

```diff
@@ -41,45 +35,35 @@ enum Operation {
 
   long getLongCardinality();
 
-
   /**
-   * set value for bsi, setValue will set each bit slice according to the input value
-   * given that we have bsi as follow
-   * ebm:  RB[1 1 1 1]
-   * slice0: RB[0 1 0 1]   -
-   * slice1: RB[1 0 1 0]  |
-   * slice2: RB[1 0 0 1]  |--bA:bit slice Array
-   * slice3: RB[0 0 0 1]   -
-   *      1 2 3 4
-   *      |
-   *      ---------------- columnId or rowId
-   *  for columnId 1, the value is 110 that is :6
-   *  for columnId 2, the value is 1 that is :1
-   *  for columnId 3, the value is 10 that is :2
-   *  for columnId 4, the value is 1101 that is :11
+   * set value for bsi, setValue will set each bit slice according to the input value given that we
+   * have bsi as follow ebm: RB[1 1 1 1] slice0: RB[0 1 0 1] - slice1: RB[1 0 1 0] | slice2: RB[1 0
+   * 0 1] |--bA:bit slice Array slice3: RB[0 0 0 1] - 1 2 3 4 | ---------------- columnId or rowId
+   * for columnId 1, the value is 110 that is :6 for columnId 2, the value is 1 that is :1 for
+   * columnId 3, the value is 10 that is :2 for columnId 4, the value is 1101 that is :11
    *
-   * @param columnId   columnId or rowId
-   * @param value    value for this columnId or rowId
+   * @param columnId columnId or rowId
+   * @param value value for this columnId or rowId
    */
   void setValue(int columnId, int value);
 
   /**
-   *
    * @param columnId columnId or rowId
    * @return the value of this columnId
    */
   Pair<Integer, Boolean> getValue(int columnId);
 
   /**
-   * setValues will batch set value for this bsi.
-   * currentMaxValue/currentMinValue are optional,it's can be compute from input value list.
-   * and avoiding bsi expend slice array capacity.
+   * setValues will batch set value for this bsi. currentMaxValue/currentMinValue are optional,it's
+   * can be compute from input value list. and avoiding bsi expend slice array capacity.
    */
   @Deprecated
-  void setValues(List<Pair<Integer, Integer>> values, Integer currentMaxValue, Integer currentMinValue);
+  void setValues(
```

Comment:
@lemire Do you feel something is wrong with some max-width parameter? (i.e. is this reformatting expected, or should I dig for maxWidth parameter?)

Reply:
That particular change does not offend me, but

Reply:
I think we  should avoid reformatting the comments too much.